Pipfile

```diff
@@ -9,38 +9,38 @@ python_version = "3"
 [packages]
 nucypher-core = "==0.13.0"
 # Cryptography
-cryptography = ">=3.2"
+cryptography = "*"
 mnemonic = "*"
 pynacl = ">=1.4.0"
 pyopenssl = "*"
 # Ethereum
-eth-abi = "<5.0.0"  # eth-ape restriction
-eth-tester = "<0.10.0,>0.9.0"  # web3[tester]
-eth-account = "<0.9,>=0.8.0"
+eth-abi = "*"
+eth-tester = "<0.10"
+eth-account = "*"
 eth-utils = "*"
 web3 = ">=6.0.0"
+atxm = ">=0.2.0"
 # Web
 flask = "*"
-hendrix = ">=4.0"
+hendrix = ">=5.0"
 requests = "*"
 mako = "*"
 # CLI
 click = ">=7.0"
 colorama = "*"
 tabulate = "*"
 # Serialization
-bytestring-splitter = ">=2.4.0"
 marshmallow = "*"
-msgpack = "*"
 # Utilities
 aiohttp = "*"
 appdirs = "*"
 constant-sorrow = ">=0.1.0a9"
 maya = "*"
 pendulum = ">=3.0.0b1"
 prometheus-client = "*"
-setuptools = "*"  # for distutils
-urllib3 = "<2,>=1.26.16"  # eth-ape
+setuptools = "*"
+urllib3 = "*"
+watchdog = "<4,>=3"
```

Comment:
Where are we using this now?

Reply:
it's a sub-dependency of both eth-ape and hendrix.  don't ask me how I know 😅

Reply:
If it's a subdependency, do we need to include it in our Pipfile? Unless the `"<4,>=3"` is particularly relevant...

Reply:
Unfortunately pipenv is unable to correctly resolve a complete version freeze in many situations, and this is a way to enforce a restriction at the top of the dependency tree that satisfies the resolver.  In other words, without this pin it's not possible to create a version lock without also fiddling around with some of the other dependency versions.  This is essentially true for most of the pins in `Pipfile` 's history that are not top-level dependencies (directly imported).

There are alternative build systems we may consider, or just simply loosen up and accept some amount of version incompatibility.